gobrew_test.go

```diff
@@ -186,7 +186,8 @@ func TestDoNotUpgradeLatestVersion(t *testing.T) {
 
 	gb := NewGoBrewDirectory(tempDir)
 
-	binaryDir := filepath.Join(gb.installDir, "bin")
+	binaryDir := filepath.Join(gb.installDir, "bin", "gobrew")
```

Comment:
I don’t understand much, why is binaryDir changing here?  This is the directory where the file will be located.

Why does the location of the binary file change?
As a result we will have:

```
~/.gobrew/bin/gobrew/gobrew
```